src/index.html

```diff
@@ -1,42 +1,39 @@
 ---
 layout: page.html
 ---
+
 <h1>Instant, themed Twitch chat overlays</h1>
 <div class="col-2">
-	<form id="overlay-builder" onsubmit="handleSubmit(event)">
+	<form id="overlay-builder">
 		<label for="channel">Channel name</label>
 		<input
 			type="text"
 			id="channel"
 			name="channel"
-			required
+			required="true"
 			autocomplete="off"
 			autocorrect="off"
 			autocapitalize="off"
 			spellcheck="false"
 		/>
-	
+
 		<label for="theme">Theme</label>
-		<select
-			id="theme"
-			name="theme"
-		>
+		<select id="theme" name="theme">
 			<option value="default">Default</option>
 			<option value="light">Light</option>
-			{%- for theme in themes.list -%}
-			{%- unless theme == "default" or theme == "light" -%}
+			{%- for theme in themes.list -%} {%- unless theme == "default" or theme
+			=="light" -%}
```

Comment:
I don't think the linter is smart enough to know that this is Liquid syntax here, but I'd prefer to keep these on their own lines if at all possible.

Reply:
after looking a bit, it appears prettier has no support for many templating languages. apart from JSX.
This was their post when they released HTML support: https://prettier.io/blog/2018/11/07/1.15.0.html#htmlvueangular

Putting this here so I can reference it later: https://stackoverflow.com/questions/50261161/how-do-i-stop-prettier-from-formatting-html-files

Since Prettier is barely configurable by choice, I don't think there is a great solution.
Do you prefer to put inline formatting ignore comments like <!-- prettier-ignore -->, or to disable Prettier from running on HTML files entirely?

Reply:
I vote let's disable Prettier on HTML entirely, since my feeling is it's unlikely we'll get HTML files that don't also include some templating.